web/www/horas/Help/credits.html

```diff
@@ -176,6 +176,11 @@ <H1 ALIGN CENTER>Credits</H1>
 
 <P ALIGN=CENTER>+++++</P>
 
+<B>Spanish</B> translation is still under development. The Spanish Vulgate translation used is the <A HREF="http://lasagradabiblia.jimdo.com/">Petisto-Torres Amat Sacred Bible</A> (La Sagrada Biblia Petisto-Torres Amat)
+For the missal translation, the <I>Misal diario latino-español y devocionario</I> (21st edition, 1962) has been used.
```

Comment:
Do you know the copyright status of this book, and in particular, do we have the necessary permissions to reproduce its texts?